#This is a wrapper message for topic mavros/global_position/compass_hdg 
#that publishes std_msgs/Float64 messages 
#The header field is added for timestamp to perform data synchonization

Header header

std_msgs/Float64 comp_hdg
